### PR TITLE
Adding instructions for local cross-compile

### PR DESCRIPTION
**Pull request by CatGarab**
Adding instructions for setting up cross-compiling in a local dev environment, rather than the SDK.

The SDK docs already have some information about cross-compiling, but I'm leaving that where it is for now. Once the dust settles (or if someone has a better idea), the information might get moved around.

### CODE REVIEW

**Comment by CatGarab** [comment]
[Cross-compile doc](https://trello.com/c/xZhUqGgz/888-cross-compile-doc)

File: docs/obc-docs/cross-compile.rst [hunk 1]
```diff
@@ -1,8 +1,80 @@
 Compiling Code for a Target OBC
 ===============================
 
-TODO: Rust and C need to use our toolchains
+In order for programs written using a compiled language (Rust, C, etc) to work on a target OBC,
```
Comment (cantino) [inline docs/obc-docs/cross-compile.rst]: *they* will

File: docs/obc-docs/cross-compile.rst [hunk 2]
```diff
@@ -1,8 +1,80 @@
...
+for reference.
+You should be able to directly copy the lines pertaining to your target OBC.
+
+Using the Toolchains
+--------------------
+
+Rust
+~~~~
+
+The new toolchains can be used by specifying the target triple you defined in the ``--target``
```
Comment (cantino) [inline docs/obc-docs/cross-compile.rst]: Maybe clearer: 

```
The new toolchains can be used by specifying the previously-defined target triple in the ``--target`` argument when running ``cargo build``.
```

File: docs/obc-docs/cross-compile.rst [hunk 3]
```diff
@@ -1,8 +1,80 @@
...
\ No newline at end of file
+The Kubos SDK comes with the cross-compile toolchains pre-installed, as well as some additional
+helper tooling.
+If you are using the SDK as your development environment, please refer to :doc:`those docs <../sdk-docs/index>`
+for more specific information.
+
+Download
+--------
+
+Below are the direct download links for the latest cross-compiling toolchains.
```
Comment (cantino) [inline docs/obc-docs/cross-compile.rst]: Worth explaining what is actually in these toolchain directories?

File: docs/obc-docs/cross-compile.rst [hunk 4]
```diff
@@ -1,8 +1,80 @@
...
+
+The new toolchains can be used by specifying the target triple you defined in the ``--target``
+argument when running ``cargo build``.
+
+For example::
+
+    $ cargo build --target arm-unknown-linux-gnueabihf
+    
+More specific information about how to set up and use a Rust project can be found in the
+:doc:`../sdk-docs/sdk-rust` doc.
```
Comment (jacoffey3) [inline docs/obc-docs/cross-compile.rst]: Wouldn't this be a link to the getting started with rust section? Or is that for dependencies? Seems like a weird place to link to.

File: docs/obc-docs/cross-compile.rst (hunk 2 above)
Comment (CatGarab) [reply docs/obc-docs/cross-compile.rst]: Thanks! That sentence was definitely giving me problems

File: docs/obc-docs/cross-compile.rst (hunk 4 above)
Comment (CatGarab) [reply docs/obc-docs/cross-compile.rst]: oh right, i forgot about that one

**Comment by CatGarab** [comment]
@plauche Ultimately yes, we should have instructions for installing `cargo kubos`, however I'd like to do that work in a different PR since the current `cargo kubos` PRs haven't landed yet

I created a card: https://trello.com/c/izDqi3mI/979-document-how-to-install-and-configure-cargo-kubos

**Pull request merged**